### ISSUE TITLE
Unlink routes from topos are not listed in activity stream

### ISSUE THREAD

**Issue by FF-Flo** [theCrag/website#4238]
I recently discovered that somebody unlinked all routes from some of my topos, so that these topos are only bare photos anymore. These edits didn't show up in the activity stream, nobody noticed, eventhough i follow the areas. I tried to do it myself -> yes, you can unlink routes in any topos and there is no mention at all, which in case of malicious users is not good. Every edit should show up in the activity stream of a route/crag.

**Comment by lordyavin** [comment]
They are not "your" topos. You created them and contributed them to the community database. But of course that is not the point.

I can't remember if this has been different in the past, so I'm considering this is a missing feature. The Topo editing component is probably quite old and might not be aligned with the stream features.

**Comment by FF-Flo** [comment]
Your are not telling anything new.

Point is anybody can unlink routes from topos with nobody noticing it. 
In this malicious case somebody unlinked ALL routes from a topo (i 
contributed) and i only noticed it by luck.

Am 29.12.2023 09:25 schrieb Kai:

**Comment by scd** [comment]
Yup this seems like a big oversight on our part. As a mitigation I have done a hotfix so that you cannot unlink a route which is drawn on the topo. Now to unlink the route you have to remove the points from the topo first. 

Whilst this does not stop the damaging edits, it does make it more transparent so at least we will see who is doing malicious edits.
